documentation/modules/rn-2.6.adoc

```diff
@@ -0,0 +1,373 @@
+== New features and enhancements
+
+This release has the following features and improvements:
+
+// .Integrate volume populators in remote migrations
+
+// Arik, please check and explain - `el9` > "Red Hat Enterprise Linux 9"?
+// Integrate volume populator for `oVirt` and OpenStack, together with `virt-v2v` on `el9` into a CDI to migrate remote {ocp-name} clusters. link:https://issues.redhat.com/browse/MTV-721[(MTV-721)]
+
+.Review and configure migrated VMs
+
+The {project-short} web console shows the name and migration status of each VM. link:https://issues.redhat.com/browse/MTV-843[(MTV-843)]
+
+//.Default virt storage class
+// change to virtual? Moved to 2.7.0
+// 
+// For requests that do not specify an explicit storage class, the `is-default-virt-class` replaces the Kubernetes default storage class. link:https://issues.redhat.com/browse/MTV-855[(MTV-855)]
+
+// Moved to 2.7.0
+//.Validate permissions on source providers
+// 
+//When migrating a VM with an Red Hat Enterprise Virtualization (RHEV) user with restricted permissions, {project-short} validates the user permissions and the migration pod reports where the process fails. link:https://issues.redhat.com/browse/MTV-273[(MTV-273)]
+
+.Validation for RHEL 6
+
+Red Hat Enterprise Linux (RHEL) 9 does not support RHEL 6 as a guest operating system, and therefore RHEL 6 is not supported in {ocp-name} Virtualization. A validation of RHEL 6 guest operating system was added. link:https://issues.redhat.com/browse/MTV-413[(MTV413)]
+
+// .Custom mapping of instance type preferences
+// 
+// {project-short} enables users to add Customer Virtual Machine Mapping, which are custom templates that are configured for specific migrations, VMs, and users. link:https://issues.redhat.com/browse/MTV-453[(MTV-453)]
+
+// [id="cross-provider-features-26_{context}"]
+=== Cross-Provider Features
+// 
+// This is a collection of new features that provide functionality across different cloud providers. link:https://issues.redhat.com/browse/MTV-802[(MTV-802)]
+// 
+.{ocp} Provider
+// any more information?
+
+This is a collection of new features for {ocp-name} (OCP) provider. link:https://issues.redhat.com/browse/MTV-739[(MTV-739)]
+
+.Block migrations from {ocp-name} versions before 4.13
+
+The `Export API` in {ocp-name} Virtualization exposes VM manifests from {ocp-name} 4.13, so source {ocp} clusters must be version 4.13 or later. link:https://issues.redhat.com/browse/MTV-734[(MTV-734)]
+
+// Pushed to 2.7
+// .Secure connections to remote {ocp-name} clusters
+// 
+// CA certificates can be specified when creating {ocp} providers. The certificate is used to validate the connection to the {ocp-name} clusters. link:https://issues.redhat.com/browse/MTV-728[MTV-728]
+
+.Source and storage endpoints are not selected automatically
+
+Source network and source storage are not selected automatically when creating an {ocp} to {ocp} plan. link:https://issues.redhat.com/browse/MTV-729[MTV729]
```

Comment:
```suggestion
```

Let us just get rid of this section as the changes are too small to highlight

Reply:
removed